src/main/java/gift/product/service/ProductService.java

```diff
@@ -46,7 +45,7 @@ public ProductResponseDto addProduct(ProductRequestDto productRequestDto) {
             product.setName(productRequestDto.getName());
             product.setPrice(productRequestDto.getPrice());
             product.setImageUrl(productRequestDto.getImageUrl());
-            return productRepository.save(product);
+            return ProductResponseDto.fromEntity(productRepository.save(product));
```

Comment:
```suggestion
            return fromEntity(productRepository.save(product));
```

여기에서도 이렇게 사용할 수 있겠네요.

Reply:
확인했습니다!